src/db/error.rs

```diff
@@ -65,6 +65,37 @@ impl DbError {
     pub fn internal(msg: &str) -> Self {
         DbErrorKind::Internal(msg.to_owned()).into()
     }
+
+    pub fn is_reportable(&self) -> bool {
+        // Yeah, this could be an "if", but this gives easy extensibility.
+        match self.inner.get_context() {
+            DbErrorKind::SpannerGrpc(ref err) => match err {
+                grpcio::Error::RpcFailure(status) => match status.status {
+                    grpcio::RpcStatusCode::ALREADY_EXISTS => false,
+                    _ => true,
+                },
```

Comment:
fair enough.

I'll comment out the block with a comment that it is an example of how to exclude a specific error, since it may not be obvious for others.